static/pdf-preview-styles.css

```diff
@@ -0,0 +1,15 @@
+@page {
+  size: A4;
+  margin-top: 15mm;
+  margin-bottom: 15mm;
+  margin-left: 10mm;
+  margin-right: 10mm;
+
+  @bottom-right {
+    content: counter(page);
+  }
+}
+
+h2 {
+  break-before: page;
```

Comment:
Page breaks may not be needed before every h2 though. I think we only need forced page breaks after the table of content.

Reply:
Noted, will improve this one later!